### PR TITLE
doc: BT service library docs clean up

### PR DESCRIPTION
**Pull request by peknis**
Mostly rewording and formatting fixes
to the BT service library docs (excluding Mesh).
Also some link fixes.

Ref: NCSDK-21324

### CODE REVIEW

File: doc/nrf/libraries/bluetooth_services/services/dfu_smp.rst [hunk 1]
```diff
@@ -54,23 +54,22 @@ For most operations, this requires a bigger MTU size than the default one.
...
-To send a command, use :c:func:`bt_dfu_smp_command`.
-The command is provided as a raw binary buffer consisting of a :c:struct:`dfu_smp_header` and the payload.
-
+To send a command, use the :c:func:`bt_dfu_smp_command` function.
+The command is provided as a raw binary buffer consisting of a :c:struct:`bt_dfu_smp_header` and the payload.
 
 Processing the response
 =======================
 
 The response to a command is sent as notification.
```
Comment (wiba-nordic) [inline doc/nrf/libraries/bluetooth_services/services/dfu_smp.rst:68]:
```suggestion
The response to a command is sent as a notification.
```

File: doc/nrf/libraries/bluetooth_services/services/fast_pair.rst [hunk 2]
```diff
@@ -82,26 +82,26 @@ Because of this requirement, the default values of the following Kconfig options
 * :kconfig:option:`CONFIG_BT_CTLR_DATA_LENGTH_MAX`
 
 .. tip::
-   In case of :ref:`nRF53 Series <ug_nrf53>`, this part of the configuration cannot be automatically updated for the network core and you must manually align it.
-   The listed options must be set on the network core to default values specified by the GFPS Kconfig options.
+   In case of the :ref:`nRF53 Series <ug_nrf53>` devices, this part of the configuration cannot be automatically updated for the network core and you must manually align it.
```
Comment (wiba-nordic) [inline doc/nrf/libraries/bluetooth_services/services/fast_pair.rst:85]:
```suggestion
   When using :ref:`nRF53 Series <ug_nrf53>` devices, this part of the configuration cannot be automatically updated for the network core and you must manually align it.
```

File: doc/nrf/libraries/bluetooth_services/services/throughput.rst [hunk 3]
```diff
@@ -17,26 +17,26 @@ The GATT Throughput Service is used in the :ref:`ble_throughput` sample.
...
 
-Throughput (0x1524)
-===================
+Throughput (``0x1524``)
+=======================
 
 Write Without Response
    * Write any data to the characteristic to measure throughput.
-   * Write 0 bytes to the characteristic to reset the metrics.
+   * Write ``0`` bytes to the characteristic to reset the metrics.
```
Comment (wiba-nordic) [inline doc/nrf/libraries/bluetooth_services/services/throughput.rst:32]: I don't think this is a correct change, and should instead be:
```suggestion
   * Write zero bytes to the characteristic to reset the metrics.
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

**Comment by kapi-no** [comment]
@alstrzebonski, can you take a quick look at the Fast Pair part?

**Review by alstrzebonski — COMMENTED** [review]
Fast Pair part approved.

File: doc/nrf/libraries/bluetooth_services/services/throughput.rst [hunk 4]
```diff
@@ -17,26 +17,26 @@ The GATT Throughput Service is used in the :ref:`ble_throughput` sample.
...
 
-Throughput (0x1524)
-===================
+Throughput (``0x1524``)
+=======================
 
 Write Without Response
    * Write any data to the characteristic to measure throughput.
-   * Write 0 bytes to the characteristic to reset the metrics.
+   * Write zero bytes to the characteristic to reset the metrics.
```
Comment (KAGA164) [inline doc/nrf/libraries/bluetooth_services/services/throughput.rst:32]: Hmm  it looks like in real it is one byte which values is not relevant, need to be written.
Maybe it is not related to this PR but it is worth to fix here

**Review by KAGA164 — APPROVED** [review]

**Comment by peknis** [comment]
> Fast Pair part approved.

@alstrzebonski, please click the green button.

**Pull request merged**